### DISCUSSION TITLE
Welcome to Cuda-Single-Page.github.io Discussions!

### DISCUSSION THREAD

**Discussion by sabbir-hossain-abir** [sabbir-hossain-abir/-Cuda-Single-Page-Portfolio-Template#3] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sabbir-hossain-abir** [comment]
Test

**Comment by ishanuzzal** [comment, marked as answer]
hello sabbir, I am a big fan of yours. I want to know from where you got this cuda template
